packages/odf/components/overview/external-systems-card/ExternalSystemsCard.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react';
+import { isExternalCluster } from '@odf/core/utils/odf';
+import { IBMFlashSystemModel } from '@odf/ibm/system-models';
+import { StatusBox } from '@odf/shared/generic/status-box';
+import { StorageClusterModel } from '@odf/shared/models/storage';
+import { resourceStatus } from '@odf/shared/status/Resource';
+import { Status } from '@odf/shared/status/Status';
+import {
+  StorageClusterKind,
+  StorageClusterPhase,
+} from '@odf/shared/types/storage';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import {
+  K8sResourceKind,
+  StatusIconAndText,
+  useK8sWatchResources,
+  WatchK8sResources,
+  YellowExclamationTriangleIcon,
+} from '@openshift-console/dynamic-plugin-sdk';
+import classNames from 'classnames';
+import * as _ from 'lodash-es';
+import { TFunction } from 'react-i18next';
+import {
+  Button,
+  ButtonVariant,
+  Card,
+  CardBody,
+  CardHeader,
+  CardProps,
+  CardTitle,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import './ExternalSystemsCard.scss';
+
+// @TODO: add IBM Scale System clusters when available.
+type AllClusters = {
+  storageClusters: StorageClusterKind[];
+  flashSystemClusters: K8sResourceKind[];
+};
+
+const resources: WatchK8sResources<AllClusters> = {
+  storageClusters: {
+    groupVersionKind: {
+      group: StorageClusterModel.apiGroup,
+      version: StorageClusterModel.apiVersion,
+      kind: StorageClusterModel.kind,
+    },
+    isList: true,
+  },
+  flashSystemClusters: {
+    groupVersionKind: {
+      group: IBMFlashSystemModel.apiGroup,
+      version: IBMFlashSystemModel.apiVersion,
+      kind: IBMFlashSystemModel.kind,
+    },
+    isList: true,
+  },
+};
+
+const getClustersStatuses = (
+  clusters: K8sResourceKind[],
+  t: TFunction
+): React.ReactNode => {
+  let [healthy, error, warning] = [0, 0, 0];
+  for (const cluster of clusters) {
+    const clusterStatus = resourceStatus(cluster);
+    if (clusterStatus === StorageClusterPhase.Ready) {
+      healthy++;
+    } else if (clusterStatus === StorageClusterPhase.Error) {
+      error++;
+    } else {
+      // We group all other statuses as warning.
+      warning++;
+    }
+  }
+
+  return (
+    <>
+      {error > 0 && (
+        <Status
+          status={StorageClusterPhase.Error}
+          title={String(error)}
```

Comment:
don't we wanna do this ??
```suggestion
          title={t('Error')}
```